business_requirement_earned_value/report/project_completion_report.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.osv import fields, osv

```

Comment:
old API?
Think you'll need to recheck this whole report

Reply:
sorry for that but the module's author not me ,I just rename the module name,and the BR not ask me fix to new API.

Reply:
@Miya609 No need to say sorry :)  

Please don't feel it as a judgement we are here to review your code and help you if we could.